libsolidity/ast/Types.h

```diff
@@ -236,8 +236,9 @@ class Type: private boost::noncopyable, public std::enable_shared_from_this<Type
 	/// of the size memoryHeadSize().
 	virtual bool hasSimpleZeroValueInMemory() const { return true; }
 	/// @returns the mobile (in contrast to static) type corresponding to the given type.
-	/// This returns the corresponding IntegerType or FixedPointType for RationalNumberType
-	/// and the pointer type for storage reference types.
+	/// This returns the corresponding IntegerType or FixedPointType for RationalNumberType,
+	/// the pointer type for storage reference types and a 160-bit unsigned IntegerType for
```

Comment:
You did not do this in the end, did you?

Reply:
Ah, sorry, didn't notice that that was still in there.